Code/Editor/Core/EditorActionsHandler.cpp

```diff
@@ -1173,6 +1419,46 @@ void EditorActionsHandler::InitializeMenus()
         }
     }
 
+    // View
+    {
+        m_menuManagerInterface->AddSubMenuToMenu(ViewMenuIdentifier, LayoutsMenuIdentifier, 100);
+        {
+            m_menuManagerInterface->AddActionToMenu(LayoutsMenuIdentifier, "o3de.action.layout.componentEntityLayout", 100);
+            m_menuManagerInterface->AddSeparatorToMenu(LayoutsMenuIdentifier, 200);
+
+            // Some of the contents of the Layouts menu are initialized in RefreshLayoutActions.
+
+            m_menuManagerInterface->AddSeparatorToMenu(LayoutsMenuIdentifier, 400);
+            m_menuManagerInterface->AddActionToMenu(LayoutsMenuIdentifier, "o3de.action.layout.save", 500);
+            m_menuManagerInterface->AddActionToMenu(LayoutsMenuIdentifier, "o3de.action.layout.restoreDefault", 600);
+        }
+        m_menuManagerInterface->AddSubMenuToMenu(ViewMenuIdentifier, ViewportMenuIdentifier, 200);
+        {
+            m_menuManagerInterface->AddActionToMenu(ViewportMenuIdentifier, "o3de.action.view.goToPosition", 100);
+            m_menuManagerInterface->AddActionToMenu(ViewportMenuIdentifier, "o3de.action.view.centerOnSelection", 200);
+            m_menuManagerInterface->AddSubMenuToMenu(ViewportMenuIdentifier, GoToLocationMenuIdentifier, 300);
+            {
+                for (int index = 0; index < 12; ++index)
```

Comment:
When that changes, I'd expect more work to be necessary honestly... But moving it out looks like a good idea, if anything to raise awareness that this code needs to be fixed too (since removing that variable would cause this to stop working)